Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -400,6 +402,12 @@ internal List<Result> RequestUninstall(string search)
 
         private void Uninstall(PluginMetadata plugin)
         {
+            
+
+            Core.Plugin.PluginManager.Settings.Plugins.Remove(plugin.ID);
+            Core.Plugin.PluginManager.AllPlugins.RemoveAll(p => p.Metadata.ID == plugin.ID);
```

Comment:
if we can do this, does this mean no restart is required?

Reply:
Well, we still cannot unload assembly unless we use the library @JohnTheGr8 has mentioned, but we can load assembly dynamically, which means that installation may not require restart.